modules/virt-attaching-vm-to-ovn-secondary-nw-cli.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_networking/virt-connecting-vm-to-ovn-secondary-network.adoc              
+
+:_content-type: PROCEDURE                                             
+[id="virt-attaching-vm-to-ovn-secondary-nw-cli_{context}"]                                   
+= Attaching a virtual machine to an OVN-Kubernetes secondary network using the CLI
+
+You can connect a virtual machine (VM) to the OVN-Kubernetes secondary network by including the network details in the VM configuration.
+
+.Prerequisites
+* You have access to the cluster as a user with `cluster-admin` privileges.
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+. Edit the `VirtualMachine` manifest to add the OVN-Kubernetes secondary network interface details, as in the following example:
++
+[source,yaml]
+----
+apiVersion: kubevirt.io/v1alpha3
+kind: VirtualMachine
+metadata:
+  name: vm-server
+spec:
+  running: true
+  template:
+    spec:
+      domain:
+        devices:
+          disks:
+            - name: containerdisk
+              disk:
+                bus: virtio
+            - name: cloudinitdisk
+              disk:
+                bus: virtio
+          interfaces:
+          - name: default
+            masquerade: {}
+          - name: flatl2-overlay
+            bridge: {}
+        machine:
+          type: ""
+        resources:
+          requests:
+            memory: 1024M
+      networks:
+      - name: default
+        pod: {}
```

Comment:
I wonder if we want to have 2 interfaces in the example - or just one. 

What do we do on the examples for other secondary networks types ?

Reply:
SR-IOV example: https://docs.openshift.com/container-platform/4.14/virt/vm_networking/virt-connecting-vm-to-sriov.html#virt-attaching-vm-to-sriov-network_virt-connecting-vm-to-sriov

Linux bridge example: https://docs.openshift.com/container-platform/4.14/virt/vm_networking/virt-connecting-vm-to-linux-bridge.html#virt-attaching-vm-secondary-network-cli_virt-connecting-vm-to-linux-bridge

Reply:
Right ! Let's follow their lead then.

Thank you.